app/views/madmin/application/index.html.erb

```diff
@@ -11,6 +11,12 @@
         </svg>
       <% end %>
     </form>
+
+    <% if policy(resource).create? %>
+      <%= link_to resource.new_path, class: "bg-white hover:bg-gray-100 text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded shadow" do %>
+        New <span class="hidden md:inline"><%= resource.friendly_name %></span>
+      <% end %>
+    <% end %>
```

Comment:
I took this back from the original gem

Reply:
What's the original gem? madmin?

Reply:
yes, sorry, from this file of madmin https://github.com/excid3/madmin/blob/14599a85f0fb687d6ad9bf30ee38db9f8b1abe3c/app/views/madmin/application/index.html.erb#L15